packages/utilities/src/utilities/smoothScroll/smoothScroll.js

```diff
@@ -31,9 +31,13 @@ const smoothScroll = (e, selector) => {
   } else {
     return null;
   }
-  document.querySelector(getSelector).scrollIntoView({
+
+  window.scroll({
+    top:
+      document.querySelector(getSelector).getBoundingClientRect().top -
+      50 +
```

Comment:
I feel like this should be an optional parameter, where the default is 0. Thoughts?

Reply:
This is a nice idea, i've implemented it on my latest commit